File: member-service/src/main/java/com/hedvig/memberservice/query/SignedMemberRepository.java
package com.hedvig.memberservice.query;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SignedMemberRepository extends JpaRepository<SignedMemberEntity, Long> {
  Optional<SignedMemberEntity> findById(long s);

  Optional<SignedMemberEntity> findBySsn(String s);
}

File: member-service/src/main/java/com/hedvig/memberservice/query/SignedMemberRepository.kt
package com.hedvig.memberservice.query

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface SignedMemberRepository : JpaRepository<SignedMemberEntity?, Long?> {
    fun findById(s: Long): SignedMemberEntity?
    fun findBySsn(s: String?): SignedMemberEntity?
}
